recipes/kneed/LICENSE

```diff
@@ -0,0 +1,12 @@
+Copyright (c) 2017, Kevin Arvai
```

Comment:
Please make a PR to the upstream repo to include the license file in the MANIFEST.in

Reply:
IIUC, you mean in the `kneed` repo?

Reply:
yup!